core/src/main/resources/error/error-classes.json

```diff
@@ -75,6 +75,23 @@
       "The value <str> (<fmt>) cannot be converted to <targetType> because it is malformed. Correct the value as per the syntax, or change its format. Use <suggestion> to tolerate malformed input and return NULL instead."
     ]
   },
+  "DATATYPE_MISMATCH" : {
```

Comment:
```suggestion
  "INPUT_DATATYPE_MISMATCH" : {
```

Reply:
This is for errors when the input data types do not meet the requirement of the function/operator

Reply:
The error class + subclass could be long. Since it will be used in docs as a header/title/ref, I don't think that adding the word INPUT can bring any benefits in error class meaning.